README.md

```diff
@@ -49,7 +48,7 @@ from github_stargazers.github import GitHub
 github = GitHub("yasoob/fb-messenger-bot")
 
 print(github.get_all_stargazers())
-print(github.is_stargazer("Robin"))
+print(github.is_stargazer("Jazzthedog"))
```

Comment:
I changed this example from a GitHub full-name to a GitHub username.

Reply:
I do agree with this change.